### DISCUSSION TITLE
Need Help in parallel Testing new feature of Laravel 8

### DISCUSSION THREAD

**Discussion by imrancoder** [laravel/framework#36093] [Q&A]
after running 
php artisan test --parallel 
it is showing 
![image](https://user-images.githubusercontent.com/68069779/106300067-01c3ad80-627c-11eb-9d28-4e7e09b4cf5d.png)

and i have updated laravel and collision as shown in image

please let me know how i can resolve it

**Comment by ahinkle** [comment]
Hey there,

Pest doesn't support parallel testing yet! Please see https://github.com/pestphp/pest/issues/120.
